docs/source/upcoming_release_notes/1199-cool-updates.rst

```diff
@@ -0,0 +1,37 @@
+1199 cool-updates
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- create a new sub module `pc.py`
+
+Device Updates
+--------------
+- in `atm.py` add `flow_meter` to `ArrivalTimeMonitor`
+- in `attenuator.py` add `flow_meter` to `AttenuatorSXR_Ladder`
+- in `attenuator.py` add `flow_meter` to `AttenuatorSXR_LadderTwoBladeLBD`
+- in `device_types.py` add `WaveFrontSensorTargetCool` , `WaveFrontSensorTargetPF2K2`
+- in `mirror.py` add flow sensor variable to `FFMirror`
+- in `wfs.py` add `flow_meter` to `WaveFrontSensorTarget`
+
+New Devices
+-----------
+- add class `PhotonCollimator` to readout `flow_switch`
+- add class `WaveFrontSensorTargetPF2K2` to readout `flow_switch`
```

Comment:
I don't understand the note here. It seems like this class exists in order to not read out any flow information? Are the notes wrong, the code wrong, or my understanding wrong?

Reply:
That's right, I could make a new class from the base class instead of what I've done here. I guess the idea is that this device will eventually use the base class and this class can be deleted. If I do it the former way, we end up with all classes inheriting a base class and no one using the base class. On the other hand, we might benefit one day from having the base class available. Probably what I am doing is more confusing than necessary. I can make a derived class and make this more paradigmatic.

Reply:
I don't understand your response here: my question isn't about the implementation, it's about the docs which don't match the implementation

Reply:
I see the confusion. Let me update the doc.

Reply:
Decided to make a new class with flow meter, so this PF2K2 class is gone and this doc is gone as well.